go/libkb/kbsig.go

```diff
@@ -252,6 +252,32 @@ func remoteProofToTrackingStatement(s RemoteProofChainLink, base *jsonw.Wrapper)
 	return nil
 }
 
+type HPrevInfo struct {
+	Seqno keybase1.Seqno
+	Hash  LinkID
+}
```

Comment:
Could you add a comment describing what the `HPrevInfo` type is?  I can't tell from the name.  I'm guessing `H` is for high?

Reply:
We're going to change this to HighSkip before merging